source/zipkin/BUILD

```diff
@@ -0,0 +1,28 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//bazel:envoy_build_system.bzl", "envoy_cc_library")
+
+envoy_cc_library(
+    name = "zipkin_lib",
+    srcs = [
+        "span_buffer.cc",
+        "span_context.cc",
+        "tracer.cc",
+        "util.cc",
+        "zipkin_core_constants.cc",
+        "zipkin_core_types.cc",
+        "zipkin_json_field_names.cc",
+    ],
+    hdrs = [
+        "span_buffer.h",
+        "span_context.h",
+        "tracer.h",
+        "tracer_interface.h",
+        "util.h",
```

Comment:
If any of these are purely internal to the library and don't need exporting for the use of other libraries, you can move them into `srcs`.